letsencrypt/plugins/standalone.py

```diff
@@ -134,18 +134,12 @@ def supported_challenges_validator(data):
 
 
 class Authenticator(common.Plugin):
-    """Standalone Authenticator.
+    """Standalone Authenticator."""
 
-    This authenticator creates its own ephemeral TCP listener on the
-    necessary port in order to respond to incoming DVSNI and SimpleHTTP
-    challenges from the certificate authority. Therefore, it does not
-    rely on any existing server program.
-
-    """
     zope.interface.implements(interfaces.IAuthenticator)
     zope.interface.classProvides(interfaces.IPluginFactory)
 
-    description = "Standalone Authenticator"
+    description = "LE Will Automatically Test Your Domain"
```

Comment:
This string should perhaps be, "Run a standalone webserver to prove that you control domains"?  Or "Use a standalone webserver to prove control of domains to the CA"?